tinyagentos/adapters/deer_flow_adapter.py

```
@@ -0,0 +1,81 @@
+            data = resp.json()
+            # The body is the final serialized channel-values. The messages list
+            # may sit at the top level or under output/values — try each.
+            messages = (
+                data.get("messages")
+                or (data.get("output") or {}).get("messages")
+                or (data.get("values") or {}).get("messages")
+                or []
+            )
+            return {"content": _extract_text(messages)}
+        return {"content": f"[{agent_name}] DeerFlow returned {resp.status_code}"}
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)